### DISCUSSION TITLE
Terima Kasih Tahun 2021 - Rahmat Agung Julians

### DISCUSSION THREAD

**Discussion by giscus** [rahmatagungj/public-rahmatagungjulians.tech#15] [Comments]
# Terima Kasih Tahun 2021 - Rahmat Agung Julians

Semua hal mengenai apa yang saya pelajari dan saya dapatkan pada tahun 2021. Banyak sekali Teknologi baru, tempat kursus dan saluran Youtube yang saya gunakan untuk belajar.

https://www.rahmatagungjulians.tech/blog/terima-kasih-tahun-2021

**Comment by rizal1919** [comment]
Keren banget
